crates/locutus-node/src/bin/node.rs

```diff
@@ -53,11 +55,21 @@ fn main() -> Result<(), DynError> {
     Ok(())
 }
 
-#[derive(clap::Parser, Clone)]
+#[derive(clap::Parser, Clone, Debug)]
 struct NodeConfig {
     /// Node operation mode.
     #[clap(value_enum, default_value_t=OperationMode::Local)]
     mode: OperationMode,
     /// Overrides the default data directory where Locutus contract files are stored.
     contract_data_dir: Option<PathBuf>,
+
+    /// Address to bind to
+    #[clap(short, long)]
```

Comment:
you can coalesce everything under `#[arg(...)]` if I am not wrong

Reply:
You are not wrong